lib/swaggerValidator/trafficValidator.ts

```diff
@@ -77,6 +133,16 @@ export class TrafficValidator {
         const payload = require(trafficFile);
         const validationResult = await this.liveValidator.validateLiveRequestResponse(payload);
         const operationInfo = validationResult.requestValidationResult?.operationInfo;
+        const swaggerFile = this.findSwaggerByOperationInfo(operationInfo);
+        if (swaggerFile !== undefined) {
```

Comment:
log an error if swaggerFile is found from cache.

Reply:
Maybe I'm not clear with above message. There's a typo for `is` which should be `isn't`.  

What I meant for `cache` is `operationSpecMapper` instead of operation cache in live validator. So, pls revert the change to query from that cache.

Reply:
opInfo is removed from validationResult, so here we need to run operation search again.